.github/workflows/ci_nightly.yml

```diff
@@ -0,0 +1,51 @@
+name: CI | Nightly
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  CI: true
+
+on:
+  schedule:
+    # Run at midnight every working day
+    - cron: '0 0 * * Mon-Fri'
```

Comment:
Do we need a nightly build? The project doesn't seem to have many dependencies so what's expected to change?

Reply:
You are right. I added a note to build only if the last CI commit is not the same as the current HEAD